### ISSUE TITLE
ready for fabrication?

### ISSUE THREAD

**Issue by KeLaiFu** [jojo-monk/Eurorack-quadrature-lfo-Yusynth#1]
Thanks for sharing your work, I have been interested in this LFO for some time, is this version of it 100% finished or is it still being developed?

**Comment by jojo-monk** [comment]
yes, I think it is ready to be manufactured, I rechecked, everything should be ok, but I haven't tested it yet...

**Comment by KeLaiFu** [comment]
Thanks for the reply, I will probably wait for you to verify it working, I plan to rework the layout a little and if I have problems with it after my adjustments, it would be very good for a noob like me to know it is my own fault :)
If it's not a huge issue, it would be very useful to add a little status note on your projects, as I am interested in doing the same with your VCO too 
Looking forward to any other work you post here, I am a big fan of the old analogue designs from yusynth.
